### ISSUE TITLE
Is there any possible way to use FontAwesome Icon while creating custom plugin ?

### ISSUE THREAD

**Issue by Rajesh-Thiyagarajan** [ckeditor/ckeditor5-react#173]
import imageIcon from '@ckeditor/ckeditor5-core/theme/icons/image.svg';
import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';

 class TemplatePreview extends Plugin {
        init() {
            const editor = this.editor;

            editor.ui.componentFactory.add('templatePreview', locale => {
                const view = new ButtonView(locale);

                view.set({
                    label: 'Template Preview',
                    icon: imageIcon,
                    tooltip: true
                });

                // Callback executed once the image is clicked.
                view.on('execute', () => {
                    
                    if (view.isOn) {
                        view.isOn = false;
                        setShowTemplatePreview(false);
                    } else {
                        view.isOn = true;
                        setShowTemplatePreview(true);
                    }
                });
                return view;
            });
        }
    }

Instead of ckeditor image icon is any possible to use fontawesome icon. Because fontawesome also svg only

**Comment by FilipTokarski** [comment]
Hi, I'm not sure I understand. You can use any `svg` file as long as you have it somewhere in your project and webpack is able to find it.

**Comment by Rajesh-Thiyagarajan** [comment]
> Hi, I'm not sure I understand. You can use any `svg` file as long as you have it somewhere in your project and webpack is able to find it.

yeah ok now i have used the svg icon only we can close this issue
